c++/src/capnp/dynamic.c++

```diff
@@ -661,6 +661,8 @@ void DynamicStruct::Builder::set(StructSchema::Field field, const DynamicValue::
           dst.set(field, src.get(field));
         }
       }
+	  
+	  return;
```

Comment:
```suggestion
      return;
```

Reply:
Yeah, noticed it too in the editor. Looked the same offline, sorry.

Reply:
Oh, you did it right as I was commenting.